src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer.java

```diff
@@ -88,7 +88,7 @@ public void run(RunConfiguration config) throws AppEngineException {
     arguments.addAll(DevAppServerArgs.get("runtime", config.getRuntime()));
     arguments.addAll(DevAppServerArgs.get("allow_skipped_files", config.getAllowSkippedFiles()));
     arguments.addAll(DevAppServerArgs.get("api_port", config.getApiPort()));
-    arguments.addAll(DevAppServerArgs.get("automatic_restart", config.getAutomaticRestart()));
+    arguments.addAll(DevAppServerArgs.get("automatic_restart", config.getAutomaticRestart(), true));
```

Comment:
@chanseokoh I think you're right that we should be setting those defaults. @meltsufin do you know if these defaults are set elsewhere?

```
paflynn@paflynn1:~/google-cloud-sdk/bin $ dev_appserver.py -h | grep -B 5 'default: True'
                        password to use when connecting to the SMTP server
                        specified in --smtp_host and --smtp_port (default: )
  --smtp_allow_tls [SMTP_ALLOW_TLS]
                        Allow TLS to be used when the SMTP server announces
                        TLS support (ignored if --smtp_host is not set)
                        (default: True)
--
                        clear the search indexes (default: False)

Task Queue API:
  --enable_task_running [ENABLE_TASK_RUNNING]
                        run "push" tasks created using the taskqueue API
                        automatically (default: True)
--
                        (default: False)
  --api_port API_PORT   port to which the server for API calls should bind
                        (default: 0)
  --automatic_restart [AUTOMATIC_RESTART]
                        restart instances automatically when files relevant to
                        their module are changed (default: True)
```